reana_client/cli/files.py

```diff
@@ -32,12 +32,14 @@
 
 from ..config import ERROR_MESSAGES, default_user
 from ..errors import FileUploadError
+from reana_client.decorators import with_api_client
 from reana_commons.utils import click_table_printer
 
 
 @click.group(
     help='All interaction related to files.')
 @click.pass_context
+@with_api_client
```

Comment:
I think we need to add `@with_api_client` in all commands in files.py

Reply:
Didn't test that, I thought that the object would be modified for next wrapped calls.. that makes it a bit inconvenient... I will add it everywhere, later we can come up with better solution. Thanks for spotting it!